challenge.md

```diff
@@ -6,19 +6,34 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+> git clone {ssh url}
+
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+> git status
+
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
+> I dont know
```

Comment:
Thanks for filling something in anyway!

The command that you would use to do this is `git log` - it will pull up a scrollable list of the commits you've made.